starcheck/data/ACABadPixels

```diff
@@ -1,99 +1,240 @@
  -512,  511,   -1,    0; ...
    -1,    0, -512,  511; ...
  -245,    0,  454,  454; ...
  -512, -512, -512, -512; ...
+ -511, -511,    9,    9; ...
+ -510, -510, -289, -289; ...
  -510, -510,  158,  158; ...
+ -510, -510,  445,  445; ...
+ -506, -506,  150,  150; ...
  -497, -497, -366, -366; ...
+ -495, -495,  207,  207; ...
+ -491, -491, -435, -435; ...
  -480, -480, -427, -427; ...
+ -476, -476,   83,   83; ...
+ -472, -472, -208, -208; ...
  -464, -464, -341, -341; ...
+ -464, -464,  -96,  -96; ...
+ -461, -461, -445, -445; ...
+ -461, -461,  292,  292; ...
+ -457, -457,  -14,  -14; ...
  -451, -451,  380,  380; ...
+ -449, -449,  268,  268; ...
  -447, -447,  244,  244; ...
+ -443, -443,  -64,  -64; ...
+ -426, -426, -159, -159; ...
+ -421, -421,   71,   71; ...
  -417, -417,  474,  474; ...
+ -416, -416,  -40,  -40; ...
  -415, -415,   28,   28; ...
  -414, -414,  -42,  -42; ...
  -411, -411,   22,   22; ...
- -402, -402, -501, -501; ...
+ -405, -405, -179, -179; ...
+ -395, -395,  462,  462; ...
+ -390, -390, -371, -371; ...
+ -390, -390, -335, -335; ...
+ -390, -390,   90,   90; ...
+ -383, -383, -247, -247; ...
+ -382, -382,  328,  328; ...
+ -381, -381, -247, -247; ...
+ -381, -381, -163, -163; ...
  -375, -375, -403, -403; ...
  -372, -372,  -40,  -40; ...
- -368, -368,  238,  238; ...
+ -364, -364,  249,  249; ...
+ -359, -359,  409,  409; ...
+ -355, -355, -132, -132; ...
  -354, -354,  387,  387; ...
  -353, -353, -486, -486; ...
+ -353, -353, -422, -422; ...
+ -352, -352,   29,   29; ...
+ -352, -352,  226,  226; ...
  -344, -344, -122, -122; ...
- -340, -340, -359, -359; ...
+ -338, -338,  -60,  -60; ...
+ -333, -333,  387,  387; ...
+ -330, -330,  -11,  -11; ...
  -316, -316,  -46,  -46; ...
  -302, -302,  237,  237; ...
  -301, -301,  391,  391; ...
+ -297, -297,  421,  421; ...
  -295, -295, -311, -311; ...
  -286, -286,  350,  350; ...
- -273, -273,  337,  337; ...
+ -283, -283, -315, -315; ...
+ -281, -281, -358, -358; ...
+ -272, -272,  -12,  -12; ...
+ -271, -271, -345, -345; ...
  -267, -267, -325, -325; ...
+ -266, -266, -124, -124; ...
+ -260, -260,  -24,  -24; ...
+ -259, -259,   59,   59; ...
  -259, -259,  122,  122; ...
  -258, -258, -491, -491; ...
  -255, -255, -251, -251; ...
- -235, -235,   97,   97; ...
+ -252, -252, -485, -485; ...
+ -252, -252,   32,   32; ...
+ -241, -241, -416, -416; ...
+ -238, -238,   93,   93; ...
+ -235, -235, -158, -158; ...
+ -231, -231, -399, -399; ...
  -231, -231,  487,  487; ...
  -217, -217,  207,  207; ...
+ -216, -216,  139,  139; ...
  -215, -215, -414, -414; ...
  -214, -214, -296, -296; ...
- -213, -213,  340,  340; ...
  -210, -210, -351, -351; ...
+ -202, -202, -280, -280; ...
+ -200, -200,  392,  392; ...
+ -198, -198,  -92,  -92; ...
+ -196, -196, -109, -109; ...
  -195, -195,  304,  304; ...
+ -194, -194, -314, -314; ...
+ -194, -194,   29,   29; ...
  -191, -191, -326, -326; ...
+ -190, -190, -380, -380; ...
+ -188, -188,  246,  246; ...
+ -186, -186, -365, -365; ...
```

Comment:
For testing, I generally spot check.  Applying this bad pixel list to MAR2717B, this bad pixel at -186, -365 spoils index 5 in obsid 18899.  This is consistent.